e2e/cypress/integration/messaging/focus_move_spec.js

```diff
@@ -100,6 +100,26 @@ describe('Messaging', () => {
         cy.get('#post_textbox').should('be.focused');
     });
 
+    it('M17450 - Focus to remain in RHS textbox each time Reply arrow is clicked', () => {
+        //# Click on Town-Square channel
+        cy.get('#sidebarItem_town-square').click({force: true});
+
+        //# Open the reply thread on the most recent post on Town-Square channel
+        cy.clickPostCommentIcon();
```

Comment:
This will work when there is a pre-existing post in the channel, not if the Town-Square channel doesn't have any posts. Could you add a simple post before this step so that the test works even when a fresh set of test-data is created? And maybe clean up at the end of the test run?